sonic_platform_base/chassis_base.py

```diff
@@ -263,12 +272,57 @@ def get_module_index(self, module_name):
         Args:
             module_name: A string, prefixed by SUPERVISOR, LINE-CARD or FABRIC-CARD
             Ex. SUPERVISOR0, LINE-CARD1, FABRIC-CARD5
+            SmartSwitch Example: DPU0, DPU1, DPU2 ... DPUX
 
         Returns:
             An integer, the index of the ModuleBase object in the module_list
         """
         raise NotImplementedError
 
+    ##############################################
```

Comment:
Do we have an API to power-on DPU, if it is down?

Reply:
Yes, under module_base  "def set_admin_state(self, up):"